eos_data_distribution/subscription.py

```diff
@@ -83,7 +83,8 @@ def _fetch_manifest_complete(self, consumer):
                 path.join(self._store_dir, 'shard', escaped_filename))
             self._shard_entries.append(
                 {'manifest_path': shard['path'], 'cache_path': shard_filename})
-            consumer = FileConsumer(shard_ndn_name, shard_filename, face=self._face)
+            consumer = FileConsumer(
+                shard_ndn_name, shard_filename, face=self._face)
```

Comment:
The changes in this commit are just formatting changes; they don’t match the commit message at all. Bad rebase/merge?